File: python/evenpal.py
#check for an evenpal number....
"""working...
success..."""
"""evenpal number is a no. which has even no. of digits and the reverse 
of it is the same as the original no."""
def check(n):
	copy1,copy2,rev,sumD=n,n,0,0
	#reverse..
	while copy2!=0:
		rev=(rev*10)+(copy2%10)
		copy2//=10
	while copy2!=0:
		sumD+=(copy2%10)
		copy2//=10
	if rev==n and sumD%2==0:
		return 0
	else:
		return 1

#driver code...
print("we will check if its an evenpal no.\nenter a no. ")
ans=check(int(input()))
if ans==0:
	print("an evenpal no.")
else:
	print("not an evenpal no.")

File: PHP/evenpal.php
<?php
//working... success....
echo "we will find evenpal no.\nenter a no.\n";
$i=(int)readline();
$sum=0;
$copy=$i;
while($copy!=0){
$sum+=($copy%10);
$copy/=10;
}
$rev=0,$copy_2=$i;
while($copy_2!=0){
	$rev=($rev*10)+($copy_2%10);
	$copy_2/=10;
}
if ($rev==$i && $sum%2==0) {
	echo $i." is a evenpal no.";
}
else{echo $i." is not a evenpal no.";}
?>